src/modules/0.0.22/aws_codedeploy/sql/after_install.sql

```diff
@@ -0,0 +1,18 @@
+ -- agent install
+-- TODO: update code for next distros in the future
+CREATE
+OR REPLACE FUNCTION generate_codedeploy_agent_install_script (region TEXT, distro TEXT) RETURNS TEXT LANGUAGE plpgsql AS $$
+BEGIN
+  CASE WHEN distro='ubuntu' THEN
+    RETURN '#!/bin/bash
+sudo apt update
+sudo apt -y install ruby-full
+sudo apt -y install wget
+cd /home/ubuntu
+wget https://aws-codedeploy-' || region || '.s3.' || region || '.amazonaws.com/latest/install
+chmod +x ./install
+sudo ./install auto
+sudo service codedeploy-agent start';
+  END case;
```

Comment:
Should we add a default case that [raises an exception](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) that it is not implemented?